src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(double a) {
+        radius = a;
+    }
+
+    @Override
+    public String getData() {
+        data = "Figure: circle, area: " + area + " sq.units, radius: " + radius
+               + " units, color: " + color;
+        return data;
+    }
+
+    @Override
+    public double getArea() {
+        double area = Math.PI * radius * radius;
+        return area;
```

Comment:
You do not need to create variable if you can simply return it
```suggestion
        return Math.PI * radius * radius;
```

Reply:
Same for all methods